src/python/install.sh

```diff
@@ -315,7 +315,7 @@ sudo_if() {
 
 install_user_package() {
     PACKAGE="$1"
-    sudo_if "$INSTALL_PATH/bin/python3" -m pip install --user --upgrade --no-cache-dir "$PACKAGE"
+    sudo_if "$CURRENT_PATH/bin/python" -m pip install --upgrade --no-cache-dir "$PACKAGE"
```

Comment:
![image](https://user-images.githubusercontent.com/24955913/189189981-bedd0885-498d-44d5-8aa2-3c240734a9ce.png)

Reply:
@jkeech as we have removed `--user` now it installs packages within the `python/bin` ☝️ and this path is known during `docker exec bash -c`

Reply:
Don't use `python` since that can be python 2.x and like on Debian bionic.  Use python3 and pip3 always.

Reply:
> Don't use python since that can be python 2.x and is on Debian buster by default along with Ubuntu bionic. Use python3 and pip3 always. If those commands aren't found, you're likely executing against Python 2.x so its a good way to verify - and python3 installs do not always set python either.

We are now linking `python3` to `python` in all scenarios except `none` case. 
Eg - [here](https://github.com/devcontainers/features/blob/main/src/python/install.sh#L280) but yeah that's a good point, switching it back to `python3`

Reply:
We are intentionally installing JupyterLab to the user site that users can install anything else with `pip install --user` and not worry about permissions issues. When Jupyter is installed by `root` to a non-user site, we see issues with Jupyter not finding modules.

Reply:
@samruddhikhandale 

> We are now linking python3 to python in all scenarios except none case.
Eg - [here](https://github.com/devcontainers/features/blob/main/src/python/install.sh#L280) but yeah that's a good point, switching it back to python3

Yeah, it's the OS version in particular that could be python 2.x.

> We are intentionally installing JupyterLab to the user site that users can install anything else with pip install --user and not worry about permissions issues. When Jupyter is installed by root to a non-user site, we see issues with Jupyter not finding modules.

That likely means it's getting installed using the wrong version of Python... probably the OS version.  If the same version of python is used in both cases, global modules will be found in addition to user ones.